### ISSUE TITLE
Make Buttons ripple effect smoother

### ISSUE THREAD

**Issue by ayoubkhan558-zz** [Daemonite/material#255]
Please Make Buttons ripple effect smoother, They are not smooth at this time.
Click on button does not feel good. Its a sudden effect.
This should be smooth.

It should be something like this.
[Material Design for Bootstrap Buttons](https://fezvrasta.github.io/bootstrap-material-design/docs/4.0/material-design/buttons/)

**Comment by djibe** [comment]
You can create issues in my own fork if you want.

**Comment by djibe** [comment]
Hi, is it OK for you ?
https://codepen.io/luncheon/pen/zYvLNvM

Looks like official ripple.

**Comment by ayoubkhan558** [comment]
> You can create issues in my own fork if you want.

Where it is link?

**Comment by ayoubkhan558-zz** [comment]
> Hi, is it OK for you ?
> https://codepen.io/luncheon/pen/zYvLNvM
> 
> Looks like official ripple.


### _ I like these ones _
http://fian.my.id/Waves/#examples
https://luncheon.github.io/ripplet.js/demo/

**Comment by djibe** [comment]
Official ones are much faster
https://material-components.github.io/material-components-web-catalog/#/component/button

This will be the reference

**Comment by djibe** [comment]
But you can customize it with Ripplet.js

**Comment by ayoubkhan558-zz** [comment]
> Official ones are much faster
> https://material-components.github.io/material-components-web-catalog/#/component/button
> 
> This will be the reference

Yes official ones are faster. But Daemonite material button effect is very sudden. It does not look good.

**Comment by ayoubkhan558** [comment]
Will you be working on this and should I work on this?

**Comment by djibe** [comment]
I think the problem comes from .btn:active background-image.
You can create a fiddle.
Or I'll try to do it during next week.

**Comment by djibe** [comment]
Here it is : https://djibe.github.io/material/docs/4.5/material/buttons/

**Comment by ayoubkhan558-zz** [comment]
@djibe Looks good now.
